linux/system/kernel.sls

```diff
@@ -53,6 +53,17 @@ linux_kernel_module_{{ module }}:
 
 {%- endfor %}
 
+{%- for kmod_config, kmod_content in system.kernel.get('kmod_config', {}).iteritems() %}
+
+/etc/modprobe.d/{{ kmod_config }}.conf:
```

Comment:
My idea is to make it possible to choose whether to use module (with specific options) or modules (to just load module).